android/KMEA/app/build.gradle

```diff
@@ -36,10 +49,6 @@ android {
     }
 
     testOptions {
-        unitTests {
-            // Allows use of a simulated Android API for tests.  (Thanks, roboelectric!)
-            includeAndroidResources = true
-        }
```

Comment:
Do they still work, or does this disconnect the unit tests?

Reply:
Yeah the unit tests still run with `./build.sh test:engine`